src/schema/server_settings.py

```diff
@@ -324,12 +326,16 @@ class ServerSettings(BaseModel):
         label="Story LLM Temperature",
         description="Temperature (creativity-factor) for the narrative generation. 0=Robot, 1=Bonkers, 0.4=Default",
         type="float",
+        min=0,
+        max=1,
     )
     default_story_max_tokens: int = SettingField(
         default=512,
         label="Story LLM Max Tokens",
-        description="Maximum number of tokens permitted during generation. 256=Default",
+        description="Maximum number of tokens permitted during generation.",
         type="int",
+        min=0,
+        max=512,
```

Comment:
I have set the max beyond 512 in certain quests. This would break my ability to continue to edit these quests. I think we should allow a much larger max here.

Reply:
👍